minecraft/protocol/packet/simulation_type.go

```diff
@@ -0,0 +1,31 @@
+package packet
+
+import "github.com/sandertv/gophertunnel/minecraft/protocol"
+
+const (
+	SimulationTypeGame byte = iota
+	SimulationTypeEditor
+	SimulationTypeTest
+	SimulationTypeInvalid
+)
+
+// SimulationType is an in-progress packet. We currently do not know the use case.
+type SimulationType struct {
```

Comment:
Is this packet sent by the client on join? If so it might just be used for the server to determine the 'simulation type'.

Reply:
I'm not sure actually, will check and see.

Reply:
Just checked, it is not sent by the client. AFAIK, it is completely unused right now.

Reply:
All right!